src/components/Calendar.tsx

```diff
@@ -91,25 +121,63 @@ export const Calendar = ({ query, queryVariables, calendarData, isLoading, navig
     [navigation, query, queryVariables, calendarData, contentContainerId]
   );
 
+  const buildListItems = useCallback(
+    (data: DateData) => {
+      const listItems = parseListItemsFromQuery(query, data, { queryVariableWithDateRange });
+
+      return listItems;
+    },
+    [query, queryVariableWithDateRange]
+  );
```

Comment:
what about refactoring to

```
const buildListItems = useCallback(
  (data: DateData) => parseListItemsFromQuery(query, data, { queryVariableWithDateRange }),
  [query, queryVariableWithDateRange]
);
```

should be the same but quicker to access, or am i wrong?

Reply:
Yes, that's exactly right. I am updating.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/34045f488b0635c5c06dffbb0f7dc8b139a98275